src/index.js

```diff
@@ -1,25 +1,19 @@
-const codependency = require('codependency');
-
 const Logger = require('./logger');
 const provisionWinston = require('./winston');
 const buildBunyanStream = require('./bunyanStream');
 
+try {
+  //  All the code below optionally configures winston if it's installed
+  const winston = require('winston');
+  const Transport = require('winston-transport');
 
-//  All the code below optionally configures winston, we need to do this
-//  here since we can only use codependency within the `package.json` `"main"`
-//  script, which gives us the correct `module`. Otherwise it errors out
-
-//  Use codependency for dynamically loading winston
-const requirePeer = codependency.register(module);
-
-//  Import winston
-const winston = requirePeer('winston', {optional: true});
-const Transport = requirePeer('winston-transport', {optional: true});
-
-//  If we have successfully loaded winston (user has it)
-//  we initialize our InsightTransport
-if (winston && Transport) {
-  provisionWinston(winston, Transport);
+  //  If we have successfully loaded winston (user has it)
+  //  we initialize our InsightTransport
+  if (winston && Transport) {
+    provisionWinston(winston, Transport);
+  }
```

Comment:
should this be outside the try/catch? if an exception happens in provisionWinston we probably shouldn't ignore it

Reply:
Ok, I'll keep the setting of the variables inside the try catch and move the provisioning outside

Reply:
Done